src/tests/functional/plugin/shared/include/behavior/compiled_model/compiled_model_base.hpp

```diff
@@ -635,6 +635,85 @@ TEST_P(OVAutoExecutableNetworkTest, AutoNotImplementedSetConfigToExecNet) {
     EXPECT_ANY_THROW(execNet.set_property(config));
 }
 
+typedef std::tuple<
+        ov::element::Type,     // Type to convert
+        std::string,           // Device name
+        ov::AnyMap             // Config
+> CompiledModelSetTypeParams;
+
+class CompiledModelSetType : public testing::WithParamInterface<CompiledModelSetTypeParams>,
+                             public OVCompiledNetworkTestBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<CompiledModelSetTypeParams> obj) {
+        ov::element::Type convert_type;
+        std::string target_device;
+        ov::AnyMap configuration;
+        std::tie(convert_type, target_device, configuration) = obj.param;
+        std::replace(target_device.begin(), target_device.end(), ':', '.');
+
+        std::ostringstream result;
+        result << "ConvertType=" << convert_type.get_type_name() << "_";
+        result << "targetDevice=" << target_device << "_";
+        if (!configuration.empty()) {
```

Comment:
Is it required?
If configuration is empty the for loop should not do any iteration.

Reply:
fixed